elements/pfe-card/src/pfe-card.scss

```diff
@@ -154,93 +165,108 @@ $variables: (
     }
   }
   &__header {
+    @include pfe-clear-spacing($properties: margin-bottom, $important: true);
+
+    display: #{pfe-local($region: header, $cssvar: Display)};
     z-index: 2;
+    
     // Declare the header background color
-    background-color: #{pfe-local($cssvar: BackgroundColor, $region: header)};
-    color:            #{pfe-local($cssvar: Color, $region: header)};
+    background-color: #{pfe-local($region: header, $cssvar: BackgroundColor)};
+    color:            #{pfe-local($region: header, $cssvar: Color)};
+    
     @include browser-query(ie11) {
       background-color: #{map-get($pfe-colors, surface--lighter)} !important;
       color: #{pfe-color(text)} !important;
     }
 
-    // Margin styles on header region
-    margin-top:    calc(#{pfe-local(PaddingTop)} * -1) !important;
-    margin-right:  calc(#{pfe-local(PaddingRight)} * -1);
-    margin-bottom: #{pfe-local(spacing--vertical)};
-    margin-left:   calc(#{pfe-local(PaddingLeft)} * -1);
-
     // Padding for the header region
     padding-top:    #{pfe-local(spacing--vertical)};
-    padding-right:  #{pfe-local(PaddingRight)};
-    padding-left:   #{pfe-local(PaddingLeft)};
     padding-bottom: #{pfe-local(spacing--vertical)};
 
+    &:not(:last-child) {
+      margin-bottom: #{pfe-local(spacing--vertical)};
+    }
+
     :host(:not([has_body],[has_footer])) & {
       margin-bottom: #{pfe-local(PaddingBottom)};
     }
 
     ::slotted([pfe-overflow~="top"]) {
-      --pfe-card__overflow--MarginTop: calc(#{pfe-local(PaddingTop)} * -1);
+      margin-top: -2rem; //IE11 fallback
+      margin-top: calc(-1 * #{pfe-local(spacing--vertical)}) !important;
     }
 
-    :host([has_body],[has_footer]) & ::slotted([pfe-overflow~="bottom"]) {
-      --pfe-card__overflow--MarginBottom: calc(#{pfe-local(spacing--vertical)} * -1);
+    ::slotted([pfe-overflow~="bottom"]) {
+      margin-bottom: -2rem; //IE11 fallback
+      margin-bottom: calc(-1 * #{pfe-local(spacing--vertical)}) !important;
     }
+  }
+  &__body {
+    @include pfe-clear-spacing($properties: margin-bottom, $important: true, $selector: last-child);
 
-    ::slotted([pfe-overflow~="bottom"]) {
-      --pfe-card__overflow--MarginBottom: calc(#{pfe-local(PaddingBottom)} * -1);
+    // Padding for the header region
+    padding-top:    #{pfe-local(PaddingTop)};
+    padding-bottom: #{pfe-local(PaddingBotom)};
+
+    &:not(:last-child) {
+      margin-bottom: #{pfe-local(spacing--vertical)};
```

Comment:
Is the `&:not(:last-child)` possibly intended for the slotted elements inside the header/body/footer regions? I'm not sure if my expectations are correct, but I'd figure that the regions would have padding, and then the children would have margin-bottom (except for the last one). 

I temporarily disabled the grid on the .demo-cards just to see the natural height & spacing, and I think the body is getting doubled up on spacing:

![image](https://user-images.githubusercontent.com/456863/79467728-1b855400-7fcc-11ea-8301-5d0202ebc286.png)

![image](https://user-images.githubusercontent.com/456863/79467468-cea17d80-7fcb-11ea-8061-2e54383a2266.png)

Reply:
Just an idea, maybe it would be cool to have an extra class sitting in the demo CSS called .natural-height or something, which just displays the cards as inline-block. We don't have to include it in the demo.html, but it would be nice to have available in web-inspector, just to spot check spacing? https://www.evernote.com/l/AA79B5wE_uNC-Jv3C5JPNvILkU0lrZfTdy4

Reply:
I'm not sure how I feel about helper classes for styling but alternatively, I just pushed up a display hook for the host element so you could do: `--pfe-card--Display: inline-block` to accomplish the same goal

Reply:
Np, the display:inline-block on the cards wont override the grid, but I can just add it manually while testing.